src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TimelineEventFactory.kt

```diff
@@ -204,4 +205,22 @@ class TimelineEventFactory {
       contextualInfo = contextualInfo,
       correlationId = correlationId,
     )
+
+  fun inductionScheduleTimelineEvent(
+    inductionSchedule: InductionSchedule,
+    eventType: TimelineEventType,
+    correlationId: UUID = UUID.randomUUID(),
+  ) =
+    TimelineEvent.newTimelineEvent(
+      sourceReference = inductionSchedule.reference.toString(),
+      eventType = eventType,
+      prisonId = "N/A",
```

Comment:
Not sure what to set this to. We don't capture it for the induction schedule so setting to N/A as it's not nullable.